elements/guzman-neilan.def

```diff
@@ -25,10 +25,7 @@ dofs:
     - normal integral moments with (lagrange,0)
 mapping: contravariant Piola
 sobolev: H1
-notes:
-  - This element is a modification of the [Bernardi&ndash;Raugel](element::bernardi-raugel) element with the facet bubbles modified to be divergence free.
```

Comment:
I think this not is no longer relevant to the new Guzman-Neilan?

Reply:
This note was not relevant before, now it is.

Reply:
You're right, I moved this note back to GN first kind